infrastructure/src/main/java/org/corfudb/infrastructure/management/HealingDetector.java

```diff
@@ -100,12 +97,15 @@ public PollReport poll(@Nonnull Layout layout,
      *
      * @return Poll Report with detected healed nodes.
      */
-    private ImmutableSet<String> pollRound(List<String> members,
-                                  Map<String, IClientRouter> routerMap,
-                                  long epoch) {
+    private PollReport pollRound(List<String> members,
+                                 Map<String, IClientRouter> routerMap,
+                                 long epoch) {
 
         Map<String, Integer> pollResultMap = new HashMap<>();
 
+        // Cluster State map for analysis.
+        Map<String, ClusterState> clusterStatusMap = new HashMap<>();
```

Comment:
clusterStateMap or change comment to match

Reply:
Done.